dbt/adapters/sqlserver/relation.py

```diff
@@ -0,0 +1,27 @@
+from typing import Optional
+
+from dataclasses import dataclass
+
+from dbt.adapters.base.relation import BaseRelation, Policy
+from dbt.exceptions import RuntimeException
+
+
+@dataclass
+class SQLServerQuotePolicy(Policy):
+    database: bool = False
+    schema: bool = False
+    identifier: bool = False
+
+
+@dataclass
+class SQLServerIncludePolicy(Policy):
+    database: bool = False
+    schema: bool = True
+    identifier: bool = True
+
+
+@dataclass(frozen=True, eq=False, repr=False)
+class SQLServerRelation(BaseRelation):
+    quote_policy: SQLServerQuotePolicy = SQLServerQuotePolicy()
+    include_policy: SQLServerIncludePolicy = SQLServerIncludePolicy()
+    quote_character: str = '"'
```

Comment:
@mikaelene this is the one thing I'm not sure about. can you forsee any issues using `"` instead of `'` to quote relation parts?

Reply:
Yeah we did, inexplicably though, the code change when I change it back to single quote. You're right that we should think about this change maybe breaking existing snapshot or incremental tables...

Reply:
I will definitely have to change this back to include `database=True` by default to make cross-database selects work.